docs/news.rst

```diff
@@ -2471,12 +2777,22 @@ First release of Scrapy.
 
 .. _AJAX crawleable urls: https://developers.google.com/webmasters/ajax-crawling/docs/getting-started?csw=1
 .. _chunked transfer encoding: https://en.wikipedia.org/wiki/Chunked_transfer_encoding
-.. _w3lib: https://github.com/scrapy/w3lib
-.. _scrapely: https://github.com/scrapy/scrapely
-.. _marshal: https://docs.python.org/2/library/marshal.html
-.. _w3lib.encoding: https://github.com/scrapy/w3lib/blob/master/w3lib/encoding.py
-.. _lxml: http://lxml.de/
 .. _ClientForm: http://wwwsearch.sourceforge.net/old/ClientForm/
-.. _resource: https://docs.python.org/2/library/resource.html
-.. _queuelib: https://github.com/scrapy/queuelib
+.. _Creating a pull request: https://help.github.com/en/articles/creating-a-pull-request
 .. _cssselect: https://github.com/SimonSapin/cssselect
```

Comment:
This is currently redirecting to `scrapy/cssselect`, but I think it would be good to update it either way.